src/mousedraghandler.cpp

```diff
@@ -253,11 +253,18 @@ MouseResizeFrame::MouseResizeFrame(MonitorManager *monitors, shared_ptr<HSFrameL
         dir = Direction::Right;
     }
     shared_ptr<HSFrame> neighbour = frame->neighbour(dir);
-    if (!neighbour || !neighbour->getParent()) {
+    if (!neighbour) {
         throw DragNotPossible("No neighbour frame in the direction of the cursor");
     }
+
     dragFrame_ = neighbour->getParent();
     auto df = dragFrame_.lock();
+
+    if (df == nullptr) {
+        // We need to check this explicitly because of -Wnull-dereference
+        throw DragNotPossible("No neighbour frame in the direction of the cursor");
```

Comment:
That message isn't quite correct is it? It is the neighbours parent that is failing here?

Reply:
This shouldn't happen anyway. The message is printed to the user by the `drag` command, so the only more helpful message is something like: `"The neighbour frame has no parent, this is a bug, please report"`.

Reply:
That makes sense, I've changed the message.

Reply:
@dnnr just updated the message, resolving this.